src/commands.rs

```diff
@@ -53,6 +53,16 @@ fn get_port() -> u16 {
         .unwrap_or(DEFAULT_PORT)
 }
 
+/// Check if ignoring all response errors
+fn ignore_all_server_errors() -> bool {
+    let ignore;
+    match env::var("SCCACHE_IGNORE_RESPONSE_ERROR") {
```

Comment:
I'd prefer SCCACHE_IGNORE_SERVER_IO_ERROR and it would be nice to have a short description of this flag in the top-level readme.